leo_cli_config.js

```diff
@@ -15,8 +9,8 @@ module.exports = {
 	}],
 	deploy: {
 		dev: {
-			stack: 'DevBus',
```

Comment:
This will break the CLI and will cause people currently using the cli to deploy this to create new a new stack